packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -703,6 +709,7 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
                     .concat(this._scaleBoxesParent.getChildMeshes())
                     .forEach((mesh) => {
                         if (pointerInfo.pickInfo && pointerInfo.pickInfo.pickedMesh == mesh) {
+                            this.onBoxOrAnchorHoverObservable.notifyObservers({ mesh });

```

Comment:
I see that other observables are doing the same thing, but - what is the point or using an object here and not directly passing the mesh?

Reply:
the observable fires when a scale box or rotation anchor is hovered, I don't think the other observables do this. This one also return the utility meshes instead of the mesh the gizmo is attached to.

Reply:
edit: observable no longer returns the util mesh as it the onHoverEnd observable would require to store it somewhere. I think just the notification is fine, similar to the isHovered variable inherited from Gizmo.

Reply:
Yes, last commit just sends an empty nofitication

Reply:
It sends an empty object:

```
this.onHoverEndObservable.notifyObservers({});
```

I just think it is redundant.

Reply:
Sorry, i feel very pedantic here :-) If you don't think it should be removed, feel free to resolve this conversation. I just don't know why we need the object to be passed, that's it :-)

Reply:
Oh, I see what you mean now, sorry coffee was wearing out. I cleared the new object and I also added types to the other observables on the last commit.